<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QTreeView" name="dtTreeView"/>
    </item>
    <item row="1" column="0">
     <widget class="QTreeView" name="btTreeView"/>
    </item>
    <item row="0" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="btLabel">
      <property name="text">
       <string>Main Behavior Tree</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="dtLabel">
      <property name="text">
       <string>Interrupt Decision Tree</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSelect_Scripts"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Behaviors</string>
    </property>
    <widget class="QMenu" name="menuInsert_Action">
     <property name="title">
      <string>Insert Action</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuInsert_Composite">
     <property name="title">
      <string>Insert Composite</string>
     </property>
     <addaction name="actionSequence"/>
     <addaction name="actionSelector"/>
    </widget>
    <widget class="QMenu" name="menuInsert_Check">
     <property name="title">
      <string>Insert Check</string>
     </property>
    </widget>
    <addaction name="menuInsert_Action"/>
    <addaction name="menuInsert_Check"/>
    <addaction name="menuInsert_Composite"/>
   </widget>
   <widget class="QMenu" name="menuDecisions">
    <property name="title">
     <string>Decisions</string>
    </property>
    <widget class="QMenu" name="menuInsert_Node">
     <property name="title">
      <string>Insert Node</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuInsert_Leaf">
     <property name="title">
      <string>Insert Leaf</string>
     </property>
    </widget>
    <addaction name="menuInsert_Node"/>
    <addaction name="menuInsert_Leaf"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuDecisions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSelect_Scripts">
   <property name="text">
    <string>Select Scripts...</string>
   </property>
  </action>
  <action name="actionSequence">
   <property name="text">
    <string>Sequence</string>
   </property>
  </action>
  <action name="actionSelector">
   <property name="text">
    <string>Selector</string>
   </property>
  </action>
  <action name="actionH">
   <property name="text">
    <string>h</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
